pkg/activator/net/revision_backends.go

```diff
@@ -159,12 +166,21 @@ func (rw *revisionWatcher) probe(ctx context.Context, dest string) (bool, error)
 		Host:   dest,
 		Path:   network.ProbePath,
 	}
-	// NOTE: changes below may require changes to testing/roundtripper.go to make unit tests passing.
-	return prober.Do(ctx, rw.transport, httpDest.String(),
+
+	options := []interface{}{
 		prober.WithHeader(network.ProbeHeaderName, queue.Name),
 		prober.WithHeader(network.UserAgentKey, network.ActivatorUserAgent),
 		prober.ExpectsBody(queue.Name),
-		prober.ExpectsStatusCodes([]int{http.StatusOK}))
+		prober.ExpectsStatusCodes([]int{http.StatusOK})}
+
+	if rw.usePassthroughLb {
+		options = append(options,
+			prober.WithHost(kmeta.ChildName(rw.rev.Name, "-private")+"."+rw.rev.Namespace),
+			prober.WithHeader("Knative-Direct-Lb", "true"))
+	}
+
+	// NOTE: changes below may require changes to testing/roundtripper.go to make unit tests passing.
```

Comment:
nit: while you're editing this line anyway.. :D

```suggestion
	// NOTE: changes below may require changes to testing/roundtripper.go to make unit tests pass.
```